src/m365/spo/commands/folder/folder-add.spec.ts

```diff
@@ -43,7 +44,8 @@ describe(commands.FOLDER_ADD, () => {
             throw addResp;
           }
           else {
-            return addResponse;
+            const isChildFolder = (opts.url as string).indexOf('%2Fabc%2Ffolder1') > -1;
```

Comment:
Wouldn't it be better to create the post stub within the function? I know that we used post stubs like this a lot in the past, but for this case, let's define only the specific stubs in the test ourselves